plugins/repository-hdfs/licenses/jetty-server-LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
@VachaShah @reta : Are there any guidelines for reviewing license content.

Reply:
I got the license from https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-server/9.4.48.v20220622 Licenses section.